src/services/api/generated/cms/graphql.ts

```diff
@@ -202,6 +200,22 @@ export type ComponentBasicImageInput = {
   style?: Maybe<Enum_Componentbasicimage_Style>;
 };
 
+export type ComponentBasicInfobox = {
```

Comment:
Det er egentlig ikke ønskelig for oss å se diff på disse genererte filene, en måte å unngå det er å tagge dem som "linguist-generated" i en fil .gitattributes, gjort blant annet i fdk-portal.

src/services/api/generated/** linguist-generated=true